doc: Fixes to the changelog (#21137)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

IPC radio firmware
------------------

* Updated:

  * The application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system in all devices that contain MRAM, such as the nRF54H Series devices.

>   * The documentation of applications and samples that use the IPC radio firmware as :ref:`companion component <companion_components>` to mention its usage when built with :ref:`configuration_system_overview_sysbuild`.

Reviewer: ```suggestion
  * The documentation of applications and samples that use the IPC radio firmware as a :ref:`companion component <companion_components>` to mention its usage when built with :ref:`configuration_system_overview_sysbuild`.
```

nRF Desktop
-----------

* Added:

  * System Power Management for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target on the application and radio cores.
  * Application configurations for the nRF54L05 and nRF54L10 SoCs (emulated on the nRF54L15 DK).
    The configurations are supported through ``nrf54l15dk/nrf54l10/cpuapp`` and ``nrf54l15dk/nrf54l05/cpuapp`` board targets.
    For details, see the :ref:`nrf_desktop_board_configuration`.
  * The ``dongle_small`` configuration for the nRF52833 DK.
    The configuration enables logs and mimics the dongle configuration used for small SoCs.
  * Requirement for zero latency in Zephyr's :ref:`zephyr:pm-system` while USB is active (:ref:`CONFIG_DESKTOP_USB_PM_REQ_NO_PM_LATENCY <config_desktop_app_options>` Kconfig option of the :ref:`nrf_desktop_usb_state_pm`).

>     The feature is enabled by default if Zephyr Power Management (:kconfig:option:`CONFIG_PM`) is enabled to prevent entering power states that introduce wakeup latency and ensure high performance.

Reviewer: ```suggestion
    The feature is enabled by default if Zephyr Power Management (:kconfig:option:`CONFIG_PM`) is enabled.
    It prevents entering power states that introduce wakeup latency and ensure high performance.
```

nRF Desktop
-----------

* Added:

>   * System Power Management for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target on the application and radio cores.

Reviewer 2: ```suggestion
  * System power management for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target on the application and radio cores.
```

  * Application configurations for the nRF54L05 and nRF54L10 SoCs (emulated on the nRF54L15 DK).
    The configurations are supported through ``nrf54l15dk/nrf54l10/cpuapp`` and ``nrf54l15dk/nrf54l05/cpuapp`` board targets.
    For details, see the :ref:`nrf_desktop_board_configuration`.
  * The ``dongle_small`` configuration for the nRF52833 DK.
    The configuration enables logs and mimics the dongle configuration used for small SoCs.
  * Requirement for zero latency in Zephyr's :ref:`zephyr:pm-system` while USB is active (:ref:`CONFIG_DESKTOP_USB_PM_REQ_NO_PM_LATENCY <config_desktop_app_options>` Kconfig option of the :ref:`nrf_desktop_usb_state_pm`).

>     The feature is enabled by default if Zephyr Power Management (:kconfig:option:`CONFIG_PM`) is enabled.

Reviewer 2: ```suggestion
    The feature is enabled by default if Zephyr power management (:kconfig:option:`CONFIG_PM`) is enabled.
```

    * Wrong header naming in :file:`provisioning_support.h` that was causing build errors when :file:`sample_reboot.h` was included in other source files.
    * An issue with an uninitialized variable in the :c:func:`handle_at_cmd_requests` function.

>     * An issue with the extremely small :kconfig:option:`CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE` Kconfig value
>       in the :file:`overlay-coap_nrf_provisioning.conf` file.

Reviewer: ```suggestion
    * An issue with a very small :kconfig:option:`CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE` Kconfig value in the :file:`overlay-coap_nrf_provisioning.conf` file.
```

PMIC samples
------------

* Added:

>   * :ref:`npm2100_one_button` sample that demonstrates how to support wake-up, shutdown, and user interactions through a single button connected to the nPM2100 PMIC.
>   * :ref:`npm2100_fuel_gauge` sample that demonstrates how to calculate the battery state of charge of primary cell batteries using the :ref:`nrfxlib:nrf_fuel_gauge`.

Reviewer: ```suggestion
  * The :ref:`npm2100_one_button` sample that demonstrates how to support wake-up, shutdown, and user interactions through a single button connected to the nPM2100 PMIC.
  * The :ref:`npm2100_fuel_gauge` sample that demonstrates how to calculate the battery state of charge of primary cell batteries using the :ref:`nrfxlib:nrf_fuel_gauge`.
```

  * Removed HID device type mapping for Development Kits.
    A Development Kit may use various HID roles (depending on configuration).

>     Assigning a fixed type per board might be misleading.

Reviewer: ```suggestion
    Assigning a fixed type for each board might be misleading.
```
